commands/order_amendment.go

```diff
@@ -90,6 +90,11 @@ func checkOrderAmendment(cmd *commandspb.OrderAmendment) Errors {
 
 	if cmd.PeggedOffset != "" {
 		isAmending = true
+		if peggedOffset, ok := big.NewInt(0).SetString(cmd.PeggedOffset, 10); !ok {
```

Comment:
I don't think this is enough - this will help for txs submitted with the wallet, but you need to also add something in market.validatePeggedOrder, where we already check for 0 but not for negative.